src/ui_parts/display/menu/zoom_menu/zoom_menu.gd

```diff
@@ -0,0 +1,54 @@
+extends HBoxContainer
+
+
+const MIN_ZOOM = 0.125
+
+const MAX_ZOOM = 64.0
+
+signal zoom_changed(zoom_level: float)
+
+signal zoom_reseted()
```

Comment:
Past time of "reset" is still "reset". Also, I see resetting doesn't emit the "zoom_changed" signal, which I find concerning but I haven't tested it.

Reply:
> Past time of "reset" is still "reset".  

Renaming to `zoom_reset_pressed` because `zoom_reset` is in use, but I'm open to better names.  

> Also, I see resetting doesn't emit the "zoom_changed" signal, which I find concerning but I haven't tested it.  

The method `zoom_reset()` change the `zoom_level` which emit `zoom_changed`.  

EDIT: I started a review? not my intention (don't know how to stop)